<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportToImage</class>
 <widget class="QDialog" name="ExportToImage">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>141</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>698</width>
    <height>141</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>698</width>
    <height>141</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Export To Image</string>
  </property>
  <property name="windowIcon">
   <iconset resource="runge.qrc">
    <normaloff>:/Runge/Resources/16x16/export1.png</normaloff>:/Runge/Resources/16x16/export1.png</iconset>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Image &amp;Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>comboBoxImageType</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Height</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEditHeight</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="labelQuality">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>JPEG &amp;Quality</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>sliderJPGQuality</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Width</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEditWidth</cstring>
   </property>
  </widget>
  <widget class="QSlider" name="sliderJPGQuality">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>62</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choose JPEG compression quality (more is better)</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxOpenAfterSave">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Open image created by default viewer</string>
   </property>
   <property name="text">
    <string>Open image a&amp;fter save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHeight">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Picture height (pixels)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxImageType">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choose image type to export to</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>596</x>
     <y>20</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditWidth">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Picture width (pixels)</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelJPEGQuality">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBoxImageType</tabstop>
  <tabstop>sliderJPGQuality</tabstop>
  <tabstop>lineEditWidth</tabstop>
  <tabstop>lineEditHeight</tabstop>
  <tabstop>checkBoxOpenAfterSave</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="runge.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportToImage</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportToImage</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderJPGQuality</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelJPEGQuality</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
